### DISCUSSION TITLE
Please report yield calculation glitches!

### DISCUSSION THREAD

**Discussion by Wilm0rien** [Wilm0rien/EveOnlineMiningYield#1] [General]
## 👋 the Community Forum is open!
* if you find that the yield calculation is not reflecting the in-game values 
or
* if you know a fitting which is able to achieve higher yield values

please post screenshots of the yield ingame values + ship fitting + skills

thanks! - Wilm0rien

**Comment by bmogensen** [comment]
What about Michis Excavation Augmentation? It goes in different slot than high wall as far as I recall and should give another 5% yield.

**Reply by Wilm0rien** [reply]
I know but it's about 1.5b so I don't think it would be a realistic to fit it. 
Further adding it to the formular whould just increase all numbers evenly and it would not contribute to the comparison of the different ships and boosts. 
please let me know if I miss something.
